src/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -57,6 +57,7 @@ public void CurrentUICulture()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Thread cultures is not honored in UWP.")]
```

Comment:
> It's totally fine to just make it no-op. We have done that in cases

I am not really worried about the performance here because this API is not used commonly and if it is used will not be called more than once. 
Some people were requesting to enable using the thread cultures in UWP too (by some config switch or some new API) so if we honored such request in the future we would use the current implementation